src/main/java/ca/ciccc/Assignment1.java

```diff
@@ -10,23 +12,24 @@
      * @return "212.0 degree Fahrenheit is equal to 100.0 in Celsius"
      */
     public static String fahrenheitToCelsius(double degree) {
-
         double celsius = (degree - 32) * ((double) 5 / 9);
         String result = String.format("%.1f degree Fahrenheit is equal to %.1f in Celsius", degree, celsius);
-
         return result;
     }
 
+
+
     /**
      * Write a function program that takes an integer in inches, converts in to meters.
      * Sample Input : 1000
      *
      * @param inches
      * @return "1000.0 inches is 25.4 meters"
      */
-    public static String inchesToMeters(int inches) {
-
-        return "";
+    public static String inchesToMeters(double inches) {
+        double meters = (inches - 975) + ((double) 4 / 10);
```

Comment:
What is 975? Argument is not always 1000. Others are same problem.

Reply:
yes, your function needs to be generic. It should work with any input value. (not only 1000)